rostime/src/time.cpp

```diff
@@ -167,6 +167,40 @@ namespace ros
     nsec = nsec_sum;
 #endif
   }
+
+  void ros_steadytime(uint32_t& sec, uint32_t& nsec)
+#ifndef WIN32
```

Comment:
This condition seems to be inverted. The exception is thrown in the Windows block.